### ISSUE TITLE
Documentation on compiling?

### ISSUE THREAD

**Issue by philgoetz** [AppleCommander/AppleCommander#30]
Can you create some documentation on how to compile this?  It appears to use gradle, which I've never used.  What do I need to type at the command line to compile it?

**Comment by a2geek** [comment]
There is a start here: https://github.com/AppleCommander/AppleCommander/blob/master/DEVELOPER.md

However, I do tend to keep things light. Let me know if there is anything else needed.

I tend to develop on a Linux desktop, so if you're on other platforms, there may be adjustments/enhancements to make.

**Comment by philgoetz** [comment]
It appears that Gradle doesn't work with newer versions of Java??

`$ ./gradlew test

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine java version from '11.0.1'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org
`

**Comment by a2geek** [comment]
Just bumped the version of Gradle to the latest:
```
$ ./gradlew --version

------------------------------------------------------------
Gradle 5.6.2
------------------------------------------------------------

Build time:   2019-09-05 16:13:54 UTC
Revision:     55a5e53d855db8fc7b0e494412fc624051a8e781

Kotlin:       1.3.41
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.14 compiled on March 12 2019
JVM:          11.0.4 (Ubuntu 11.0.4+11-post-Ubuntu-1ubuntu218.04.3)
OS:           Linux 4.15.0-64-generic amd64

```

**Comment by philgoetz** [comment]
It works now!
I suggest adding a note to DEVELOPER.md saying that the built .jar files are in /build/libs, and also that you should use the win64 version if you have Windows 64-bit, and also that the .jar filename will be set to the name of the current working directory.

Thanks!

**Comment by philgoetz** [comment]
I tried to compile the new version, to see if the [Windows bug](https://github.com/AppleCommander/AppleCommander/issues/35) has been fixed, and I can't figure out how to compile it.
I didn't keep notes on how I did it last time, but it seems to use a different process, since ./gradlew test doesn't do anything except print informational messages, and there is no build/libs directory.
Please put some instructions somewhere on how to compile AppleCommander.

**Comment by a2geek** [comment]
The updates for #35 are included in AppleCommander 1.6.0, and you can download it here: https://github.com/AppleCommander/AppleCommander/releases/tag/v1-6-0

As far as compiling, it's the Gradle ecosystem. Notes here: https://github.com/AppleCommander/AppleCommander/blob/master/DEVELOPER.md

Gradle is here: https://gradle.org/. This is a big topic and I don't think AC should be documenting how to use or setup Gradle. If details are missing, then certainly, that should be remediated.

**Comment by a2geek** [comment]
Closing due to inactivity.
